### ISSUE TITLE
Speedbrake indicator

### ISSUE THREAD

**Issue by NikolaiVChr** [NikolaiVChr/f16#168]
legoboyvdlp: Speedbrake indicator slightly incorrect:
- should show CLOSE when closed
- should show diagonals in-transit or with no electrical power
- should show the dots when opened

Also, in the real viper, the speedbrakes may be opened incrementally - don't think this is possible in ours. Is this just a "simism" for ease of use that we don't have this?  Because if so, dots means > 2 degrees open.

**Comment by NikolaiVChr** [comment]
![image](https://user-images.githubusercontent.com/3764194/79050916-723a0900-7c2d-11ea-92dd-dfc32e35af20.png)

**Comment by NikolaiVChr** [comment]
Right now its just controlled using:
```xml
 <animation>
  <type>rotate</type>
  <object-name>SPEED_BRAKE_drum</object-name>
  <property>surface-positions/speedbrake-pos-norm</property>
  <factor>95</factor>
  <center>
   <x-m>-0.094</x-m>
   <y-m>-0.424</y-m>
   <z-m>0.001</z-m>
  </center>
  <axis>
   <x>0</x>
   <y>-1</y>
   <z>0</z>
  </axis>
 </animation>
```

**Comment by NikolaiVChr** [comment]
![image](https://user-images.githubusercontent.com/3764194/79051178-08bafa00-7c2f-11ea-95e7-d9476397b3a6.png)

**Comment by NikolaiVChr** [comment]
Part of design document for indicator
![image](https://user-images.githubusercontent.com/3764194/79051195-212b1480-7c2f-11ea-82de-11cd42186cde.png)

**Comment by NikolaiVChr** [comment]
fixed with d876bd7f491a582cb18c721ca6bead77d0ae8112
